### ISSUE TITLE
Streams will not start

### ISSUE THREAD

**Issue by msmittag** [pigskin/kodi-gamepass#431]
Hi folks,

so I am using a Raspberry Pi3 and run LibreELEC 9.2.8 with Kodi 18.
I installed:
-	iso8601
-	m3u8
-	Requests 2.x
-	kodi-gamepass-master.zip
I am able to login to the app OK with my gamepass Europe subscription but no stream will start. No matter if I choose full game, highlights or compressed game. Same with NFL network.

Any hints?

Cheers
Michael

**Comment by rschloeg** [comment]
Same for me on a Raspberry Pi4. After choosing a game I get the popup which allows me to choose the resolution - and that's it. No video and no error message popping up.

**Comment by jm-duke** [comment]
Yeah, non-adaptive streams have been broken for a while.

Iirc, LibreELEC does not ship with the inputstream adaptive addon. Please try installing the "InputStream Adaptive" add-on and configure adaptive streams in the settings dialog of the gamepass addon.

**Comment by msmittag** [comment]
Hi Thomas,
oh shoot... Now I remember I had to do something like that say 4 years ago but did'nt when I updated Operation system and Kodi.Will try. The add-on is in the repository?
CheersMichael
 
 
 
-----Original Message-----

**Comment by jm-duke** [comment]
Yes, it is.

**Comment by rschloeg** [comment]
Thank you very much for the reply to @jm-duke.

Input Stream Adaptive was already installed on my RPi and before commenting here I did a try first with deactivated and then with activated Adaptive Streams in Gamepass settings. Both did **not** work initially.
But your answer made think again and I remembered from years ago that some setting changes needed a reboot. And indeed, after a reboot it worked fine.

So a hint for everybody: Reboot after activating adaptive streams in the Gamepass settings - at least on a Raspberry Pi.
